README.md

```diff
@@ -37,8 +37,35 @@ To check whether any format violation exists, run `spotlessCheck` gradle task. I
 
 Both plugins are enabled on gradle's `compileJava` task.
 
-## References
+### Testing
+
+This project uses separate directories for unit and integration tests:
+
+- [unit test](src/test)
+- [integration test](src/integrationTest)
+
+#### Gradle Tasks
+
+##### Unit Test
+```console
+./gradlew clean test
+```
 
+##### Integration Test
+```console
+./gradlew clean integrationTest
+```
+
+Integration tests require a MongoDB deployment. You may change the default [MongoDB connection string](https://www.mongodb.com/docs/manual/reference/connection-string/) as below, in [hibernate.properties](src/integrationTest/resources/hibernate.properties):
+
+```properties
+jakarta.persistence.jdbc.url={your_mongodb_connection_string}
```

Comment:
I also changed the above properties using placeholder. Repeating existing property seems not to make sense (which is a moving target).